components/calendar/calendar.js

```diff
@@ -323,6 +324,7 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 		const weekdayHeaders = calendarData.daysOfWeekIndex.map((index) => html`
 			<th
 				abbr="${calendarData.descriptor.calendar.days.long[index]}"
+				aria-label="${calendarData.descriptor.calendar.days.long[index]}"
```

Comment:
In NVDA without this for some days of the week it will say the abbreviated version (e.g., `Mon` instead of `Monday`)